data/policies/filecon.cas

```diff
@@ -6,6 +6,9 @@ resource foo {
 	file_context("/etc/somesymlink", [lnk_file], this);
 	file_context("/var/somepipe", [fifo_file], this);
 	file_context("/var/somesocket", [sock_file], this);
+	file_context("/bin/some_bin", [file], system_u:object_r:foo:s0);
+	file_context("/bin/some_bin", [file], system_u:object_r:foo:s0-s0);
+	file_context("/bin/some_bin", [file], system_u:object_r:foo:s0-s0:c0.c255);
```

Comment:
I am guessing at some point we want to at least warn if the paths are identical with different contexts for file cons.  I would update the paths to be different here just to future proof this.

Reply:
Good catch.  It looks like right now we silently drop the duplicates.  I think we should error there.  I filed #131 to investigate that.